.github/workflows/qt_viz_tests.yml

```diff
@@ -167,10 +151,6 @@ jobs:
         run: |
           pip uninstall -yq PyQt5 PyQt5-Qt5 PyQt5-sip
           pip install ${{ matrix.qt }}
-          echo "::set-env name=PYQTGRAPH_QT_LIB::${{ matrix.qt }}"
```

Comment:
FYI @GuillaumeFavelier I removed this as it should not be necessary -- PyQtGraph says it first looks to see which APIs have been imported, and since our `qtpy` import is above the `pyqtgraph` import, the resulting qt api should be respected